arviz/data/inference_data.py

```diff
@@ -11,19 +11,53 @@
 
 
 class InferenceData:
```

Comment:
It would be nice to add the `Attributes` section to the docstring for this class, since they are so important.

Am I right in thinking that the "standard" groups are `posterior`, `prior`, and `posterior_predictive`, but that the caller can add whatever groups they want?  Maybe explain this in the docstring (if you are documenting the `groups` attribute).

Reply:
I was intentionally leaving the attributes and groups out until we have had the ArviZ devs meeting and discussed about them, I'd rather add them in another PR (hopefully already including some way to store multiple log likelihood, out of sample predictions and the predictors used to get these out of sample predictions and any other modification to the schema).